llvm/include/llvm/CodeGen/MachineFunctionPass.h

```diff
@@ -18,6 +18,7 @@
 #ifndef LLVM_CODEGEN_MACHINEFUNCTIONPASS_H
 #define LLVM_CODEGEN_MACHINEFUNCTIONPASS_H
 
+#include "llvm/CodeGen/DroppedVariableStats.h"
```

Comment:
This include is mindbogglingly expensive (2% on a single-backend build). Please change your design in a way that avoids pulling in the whole kitchen sink into MachineFunctionPass.h...